xml/common_intro_typografie_i.xml

```diff
@@ -11,20 +11,59 @@
  <title>Documentation Conventions</title>
 
  <info>
-<dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
-    <dm:maintainer>fs</dm:maintainer>
-    <dm:status>editing</dm:status>
-    <dm:deadline/>
-    <dm:priority/>
-    <dm:translation>no</dm:translation>
-    <dm:languages/>
-</dm:docmanager>
-</info>
-<para>
-  The following typographical conventions are used in this manual:
+  <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+   <dm:maintainer>fs</dm:maintainer>
+   <dm:status>editing</dm:status>
+   <dm:deadline/>
+   <dm:priority/>
+   <dm:translation>no</dm:translation>
+   <dm:languages/>
+  </dm:docmanager>
+ </info>
+
+ <para>
+  The following admonition elements and typographical conventions are used
```

Comment:
Good point, changed it to 'notices' now (because not all of them are for warning purposes).